code/datums/components/deployable.dm

```diff
@@ -41,39 +48,42 @@
 /datum/component/deployable/proc/examine(datum/source, mob/user, list/examine_list)
 	SIGNAL_HANDLER
 
-	examine_list += span_notice("[source.p_they()] look[source.p_s()] like [source.p_they()] can be deployed into \a [deployed_name].")
+	examine_list += span_notice("It can be used <b>in hand</b> to deploy into \a [deployed_name].")
 
 /datum/component/deployable/proc/on_attack_hand(datum/source, mob/user, location, direction)
 	SIGNAL_HANDLER
 	INVOKE_ASYNC(src, PROC_REF(deploy), source, user, location, direction)
 
 /datum/component/deployable/proc/deploy(obj/source, mob/user, location, direction) //If there's no user, location and direction are used
-	var/obj/deployed_object //Used for spawning the deployed object
-	var/turf/deploy_location //Where our deployed_object gets put
-	var/new_direction //What direction do we want our deployed object in
-	if(user)
-		if(!ishuman(user))
-			return
+	/// The object we are going to create
+	var/atom/deployed_object
+	/// The turf our object is going to be deployed to
+	var/turf/deploy_location
+	/// What direction will the deployed object be placed facing
+	var/new_direction
```

Comment:
```suggestion
	// The object we are going to create
	var/atom/deployed_object
	// The turf our object is going to be deployed to
	var/turf/deploy_location
	// What direction will the deployed object be placed facing
	var/new_direction
```
Minor but I think dmdocs in the middle of a proc will confuse it into thinking these apply to the next proc